### PR TITLE
Update disk installation docs to reflect Solus 4.1

### PR DESCRIPTION
**Pull request by sethfl**
Added descriptions for LVM and encryption to reflect new options in the Solus 4.1 installer

## Description

<Info on what this pull request adds>

### Submitter Checklist

- [x] Updated the "lastmod" portion at the top of any modified Markdown files.
- [ ] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

**Comment by DataDrake** [comment]
With the way this is written, it will be misunderstood by the end-user. The installation options are:
1. Use whole disk
  1. Normal root partition
  2. LVM managed root partition
    - (Optional) LUKS encryption of everything but `ESP` (UEFI) or `/boot` (BIOS)
2. Install along side existing (normal root partition)
3. Assign mountpoints (Manual partition creation and assignment)

**Comment by sethfl** [comment]
updated it a little bit, and synced it with upstream

File: installation/disks/en.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 +++
 # Disks
 
 Solus provides multiple ways you can install onto your system.
 
-1. Solus can take up the entire drive.
+1. Solus can take up the entire drive, using traditonal root partitions, or with LVM-managed root partitions.
 2. You can install Solus next to your existing operating system, such as Windows. *If you have multiple operating systems 
-installed, we will choose the largest option.*
+installed, we will choose the largest option.*
```
Comment (JoshStrobl) [inline installation/disks/en.md:11]: Let's put this on line 10, not sure why there was a line break there.

**Comment by JoshStrobl** [comment]
LGTM, thanks!

**Pull request merged**